packages/phoenix-evals/pyproject.toml

```diff
@@ -30,6 +30,8 @@ dependencies = [
   "pystache",
   "pydantic>=2.0.0",
   "glom",
+  "openinference-semantic-conventions",
```

Comment:
can you set a high min version here? Otherwise it's scary

Reply:
sounds good